data.sql

```diff
@@ -17,4 +17,30 @@ VALUES ('Charmander', '2020-02-08', 0, false, -11),
 	   ('Angemon', '2005-06-12', 1, true, -45),
 	   ('Boarmon', '2005-06-07', 7, true, 20.4),
 	   ('Blossom', '1998-10-13', 3, true, 17),
-	   ('Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+	   ('Ditto', '2022-05-14', 4, true, 22);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34),
+		('Jennifer Orwell', 19),
+		('Bob', 45),
+		('Melody Pond', 77),
+		('Dean Winchester', 14),
+		('Jodie Whittaker', 38);
+
+INSERT INTO species (name)
+VALUES ('Pokeman'),
+	   ('Digimon');
```

Comment:
👍 Great work on your SQL queries! However, there's a small adjustment needed to meet the project requirements:

- [ ] As per the project requirement, you should insert the following data into the "species" table: 'Pokemon' and 'Digimon.' I noticed a small typo where you inserted 'Pokeman' instead of 'Pokemon.' Please update your code to insert the data as 'Pokemon' and 'Digimon' to match the project requirements.

- [ ] Additionally, kindly consider running all the queries in the `queries.sql` file and update the screenshots in the PR Description if necessary to ensure your PR aligns perfectly with the project's requirements. Keep up the excellent work! 👍📋🔍📷